python/ray/data/tests/test_stats.py

```diff
@@ -223,6 +229,35 @@ def test_large_args_scheduling_strategy(ray_start_regular_shared):
 * Output size bytes per block: N min, N max, N mean, N total
 * Output rows per task: N min, N max, N mean, N tasks used
 * Tasks per node: N min, N max, N mean; N nodes used
+
+Operator N MapBatches(dummy_map_batches): {EXECUTION_STRING}
+* Remote wall time: T min, T max, T mean, T total
+* Remote cpu time: T min, T max, T mean, T total
+* Peak heap memory usage (MiB): N min, N max, N mean
+* Output num rows per block: N min, N max, N mean, N total
+* Output size bytes per block: N min, N max, N mean, N total
+* Output rows per task: N min, N max, N mean, N tasks used
+* Tasks per node: N min, N max, N mean; N nodes used
+"""
+    )
+
+    # Try with verbose logs
```

Comment:
to minimize test complexity, can we keep one main ray data logic chunk, change `verbose_stats_logs` value based on `pytest.parametrize`, use the same base expected output stats string for both cases, and add the extra metrics bit only if `verbose_stats_logs` is True?